README.md

```diff
@@ -102,7 +102,47 @@ A preview of the VDK CLI commands:
 
 
 # Getting Started
-Installing VDK is a simple pip command. See the [Getting Started](https://github.com/vmware/versatile-data-kit/wiki/getting-started) guide to install VDK and create a data job.
+
+## Vdk-core and CLI
```

Comment:
Put the term"VDK SDK" or Quickstart VDK SDK even.

 vdk-core is an implementation detail . Nobody is going to install vdk-core as by itself it's just a plugin framework with some basic pre-built plugins. It's all the other plugins that the SDK really useful.